ddtrace/internal/_encoding.pyx

```diff
@@ -732,8 +740,15 @@ cdef class MsgpackEncoderV05(MsgpackEncoderBase):
         ret = msgpack_pack_int32(&self.pk, _ if _ is not None else 0)
         if ret != 0: return ret
 
-        ret = msgpack_pack_map(&self.pk, len(span._meta) + (dd_origin is not NULL))
+        ret = msgpack_pack_map(&self.pk, len(span._meta) + (dd_origin is not NULL) + (i == 0))
         if ret != 0: return ret
+
+        if i == 0:
+            ret = self._pack_string("language")
+            if ret != 0: return ret
+            ret = self._pack_string("python")
+            if ret != 0: return ret
```

Comment:
we should try to mimic the dd origin code below.

the v0.5 encoder uses a string table to store key names and values. so we actually end up packing in integers instead of strings/bytes here.

since `language` and `python` are going to always show up in every single payload, we can pre-create the string table with entries for `language` and `python`, then we can just pack in the hard coded string values here (see the origin key below)